package.json

```diff
@@ -19,7 +19,7 @@
     "build:esnext": "tsc",
     "build:es5": "rollup -c && cp dist/esnext/uri.d.ts dist/es5/uri.all.d.ts && npm run build:es5:fix-sourcemap",
     "build:es5:fix-sourcemap": "sorcery -i dist/es5/uri.all.js",
-    "build:es5:min": "uglifyjs dist/es5/uri.all.js --support-ie8 --output dist/es5/uri.all.min.js --in-source-map dist/es5/uri.all.js.map --source-map uri.all.min.js.map --comments --compress --mangle --pure-funcs merge subexp && mv uri.all.min.js.map dist/es5/ && cp dist/es5/uri.all.d.ts dist/es5/uri.all.min.d.ts",
+    "build:es5:min": "uglifyjs dist/es5/uri.all.js --output dist/es5/uri.all.min.js --source-map \"content='dist/es5/uri.all.js.map',filename='uri.all.min.js.map'\" --comments --mangle --compress \"pure_funcs=['merge', 'subexp']\" && cp dist/es5/uri.all.d.ts dist/es5/uri.all.min.d.ts",
```

Comment:
As described here: https://github.com/mishoo/UglifyJS

Compared to the old version: https://github.com/mishoo/UglifyJS/tree/v2.8.14